workflows/utilities/wf_read_QC_trim_se.wdl

```diff
@@ -82,6 +84,16 @@ workflow read_QC_trim_se {
         midas_db = midas_db
     }
   }
+  if ("~{workflow_series}" == "theiaprok") {
+    if (call_kraken) {
+      call kraken.kraken2_standalone {
+        input:
+          samplename = samplename,
+          read1 = read1_raw,
+          kraken2_db = select_first([kraken_db])
```

Comment:
These things always throw me off. Just to make sure I have this right: since the `kraken2_db` workflow variable is optional, but the task variable is obligate, we have to use this select_first() function, otherwise the WDL syntax check will prevent it from importing into Terra--is that correct?

Reply:
yes! it even fails our internal CI with miniwdl check!